vertx-quickstart/src/test/java/org/acme/vertx/FruitsEndpointTest.java

```diff
@@ -20,40 +20,24 @@
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.core.IsNot.not;
 
-import org.junit.jupiter.api.Test;
-
 import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
 
 @QuarkusTest
 public class FruitsEndpointTest {
 
     @Test
     public void testListAllFruits() {
-        //List all, should have all 3 fruits the database has initially:
-        given()
-                .when().get("/fruits")
-                .then()
-                .statusCode(200)
-                .body(
-                        containsString("Orange"),
-                        containsString("Pear"),
-                        containsString("Apple"));
-
-        //Delete the Orange:
-        given()
-                .when().delete("/fruits/1")
-                .then()
-                .statusCode(204);
-
-        //List all, Orange should be missing now:
-        given()
-                .when().get("/fruits")
-                .then()
-                .statusCode(200)
-                .body(
-                        not(containsString("Orange")),
-                        containsString("Pear"),
-                        containsString("Apple"));
+        // List all, should have all 3 fruits the database has initially:
+        given().when().get("/fruits").then().statusCode(200).body(containsString("Orange"), containsString("Pear"),
+                containsString("Apple"));
+
+        // Delete the Orange:
+        given().when().delete("/fruits/1").then().statusCode(204);
+
+        // List all, Orange should be missing now:
+        given().when().get("/fruits").then().statusCode(200).body(not(containsString("Orange")), containsString("Pear"),
+                containsString("Apple"));
```

Comment:
Frankly, this is not an improvement.

The tests are made to be fully understandable and this definitely doesn't help understanding what's going on.

@cescoffier WDYT?

Reply:
i didn’t do anything special here - just used quarkus default formatting style. 

I can try and tweak the format using https://stackoverflow.com/questions/18324017/how-to-set-eclipse-code-formatter-to-support-fluent-interfaces or we can to be safe utilize
```
// @formatter:off
...
// @formatter:on
```

to have ful control

Reply:
so the setting in SO works in eclipse giving me this:

```
   public void testListAllFruits() {
        // List all, should have all 3 fruits the database has initially:
        given().when()
               .get("/fruits")
               .then()
               .statusCode(200)
               .body(containsString("Orange"),
                       containsString("Pear"),
                       containsString("Apple"));

        // Delete the Orange:
        given().when()
               .delete("/fruits/1")
               .then()
               .statusCode(204);

        // List all, Orange should be missing now:
        given().when()
               .get("/fruits")
               .then()
               .statusCode(200)
               .body(
                       not(containsString("Orange")),
                       containsString("Pear"),
                       containsString("Apple"));
    }
```

but looks like https://code.revelc.net/formatter-maven-plugin does not support that setting ..at least it seem to ignore it ;/ . investigating.

Reply:
hmm - seems like the formatter completely ignores the formatter when no config in the pom.xml....explains why its not formatting but sucks if true ;/

Reply:
okey so there is unfortunately no way to easily have the formatter pick up the shared resources via mvn unless we add formatter settings to each plugin.

as work around i've made it so it points to the format via ../quarkus.

i've also committed code is it would look if change formatter to automatically put a linebreak and indent call chains per nested column.

This would require a change to the share format to work.